docs/user/reporting/reporting-csv-troubleshooting.asciidoc

```diff
@@ -0,0 +1,91 @@
+It is recommended that you use CSV reports to export moderate amounts of data only.
+The feature enables analysis of data in external tools, but it's not intended for bulk export or to backup {es} data.
+If you need to export more than 250 MB of CSV, rather than increasing <<reporting-csv-settings,`xpack.reporting.csv.maxSizeBytes`>>, use
+filters to create multiple smaller reports or extract the data you need directly from {es}.
+
+The following deployment configurations may lead to failed report jobs or incomplete reports:
+
+* Any shard needed for search is unavailable.
+* Data is stored on slow storage tiers.
+* Network latency between nodes is high.
+* {ccs-cap} is used.
+
+To export large amounts of data, use {es} APIs directly.
+Check out the {ref}/point-in-time-api.html[Point in time API] or {ref}/sql-rest-format.html#_csv[SQL with CSV response data format].
+============
+
+[float]
+[[reporting-troubleshooting-csv-configure-scan-api]]
+=== Configuring CSV export to use the scroll API
+
+The Kibana CSV export feature collects all of the data from Elasticsearch by using multiple requests to page
+over all of the documents.
+Internally, the feature uses the {ref}/point-in-time-api.html[Point in time API and
+`search_after` parameters in the queries] to do so.
+There are some limitations related to the point in time API:
+
+1. Permissions to read data aliases alone will not work: the permissions are needed on the underlying indices or datastreams.
+2. In cases where data shards are unavailable or time out, the export will be empty rather than returning partial data.
+
+Some users may benefit from using the {ref}/paginate-search-results.html#scroll-search-results[scroll API], an
+alternative to paging through the data.
+The behavior of this API does not have the limitations of point in time API, however it has its own limitations:
+
+1. Search is limited to 500 shards at the very most.
+2. In cases where the data shards are unavailable or time out, the export may return partial data.
+
+If you prefer the internal implementation of CSV export to use the scroll API, you can configure this in
+`kibana.yml`:
+
+[source,yml]
+-------------------------------------------
+xpack.reporting.csv.scroll.strategy: scroll
+-------------------------------------------
+
+For more details about CSV export settings, go to <<reporting-csv-settings>>.
+
+[float]
+[[reporting-troubleshooting-csv-socket-hangup]]
+=== Socket hangups
+
+A "socket hangup" is a generic type of error meaning that a remote service (in this case Elasticsearch or a proxy in Cloud) closed the connection.
+Kibana can't foresee when this might happen and can't force the remote service to keep the connection open.
+One thing it can do is try the advice @Dosant gave to lower the size of results that come back in each request or increase the amount of time the remote services will
+allow to keep the request open.
+Considering the evidence in the logs and the results of they are trying to get,
```

Comment:
Unless we can give advice on what to look for in the logs, I think this should be removed
```suggestion

```